lib/template-loader.php

```diff
@@ -195,6 +195,7 @@ function gutenberg_find_template( $template_file ) {
 
 		$_wp_current_template_content = $current_template_post->post_content;
 	}
+	$_wp_current_template_content = $current_template_post->post_content;
```

Comment:
I had to add this line to get templates working. Not sure what I'm doing wrong.

Reply:
> I had to add this line to get templates working. Not sure what I'm doing wrong.

See also #18877. You may need to rebase.